src/navability/entities/factor/distributions.py

```diff
@@ -37,9 +67,10 @@ def marshal(self, data, **kwargs):
 
 @dataclass
 class FullNormal(Distribution):
-    _type: str = "IncrementalInference.PackedFullNormal"
+    # TODO: Remove the default initializers.
```

Comment:
I actually think default could or should just be perfect information? I like the idea of being able to show code without context. Just like we do in the story we start with needing to know very little. Currently the problem I have with Caesar is I have to know SO much just to get code that runs from top to bottom.

Reply:
So problem with perfect information is it'll cause a convergence issue (it's like, "I know i'm here") - @dehann can you weigh in here?

Fair on the amount of information needed. For distributions, I think defaults are tricky - like Normal's is generally (0,1), Categorical is (0.5,0.5), Uniform is (-1, 1) - but a real user would have an idea of what their distribution should be so I'm hesitant to use those.

Reply:
If by perfect info, you mean a dirac impulse rather than normal bell curve, hence variance goes to 0... We would not be able to run computations on that.  It would dominate the computations with (as Sam said) "I know I'm here".

> I like the idea of being able to show code without context. Just like we do in the story we start with needing to know very little.

What we perhaps should do is use the defaults as suggested, Normal(0,1), etc.  I have been adding those into the Distribution contracts so the actual quantities are optional:
https://github.com/JuliaRobotics/IncrementalInference.jl/blob/master/src/Serialization/entities/SerializingDistributions.jl

Jim, does that help answer what you're asking?

> Currently the problem I have with Caesar is I have to know SO much just to get code that runs from top to bottom.

Yeah, agreed we have to overcome this else adoption will be slower.

Reply:
Done. Going to add an issue about default initialization of distributions as a separate issue.